MapboxVisionAR/Render/ARScene/ARNode/ARNodeProtocol/ARNode+Extension.swift

```diff
@@ -0,0 +1,96 @@
+import simd
+
+/// Properties common to the AR nodes which will be displayed in `ARScene`.
+protocol ARNode: Node {
+    /// Type of node.
```

Comment:
Do we need such an obvious comments?

Reply:
The answer is yes. I tried to did comments on all types/pieces of types so that's the reason why even such obvious code has comment. It's a rare case and I think it definitely will be a bit weird if we have comments for all fields except the only one. You can propose more "non-obvious" comment anytime.